site/_docs/reference.md

```diff
@@ -1291,6 +1291,7 @@ Not implemented:
 | LOCALTIMESTAMP(precision) | Returns the current date and time in the session time zone in a value of datatype TIMESTAMP, with *precision* digits of precision
 | CURRENT_TIME              | Returns the current time in the session time zone, in a value of datatype TIMESTAMP WITH TIME ZONE
 | CURRENT_DATE              | Returns the current date in the session time zone, in a value of datatype DATE
+| CURRENT_DATETIME          | Returns the current datetime in the session time zone, in a value of datatype TIMESTAMP
```

Comment:
Not sure if this makes sense, also when I added CURRENT_DATETIME to reserved words I broke `DocumentationTest.testGenerateKeyWords`